svgjs.resize/svgjs.resize.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for svgjs.resize
+// Project: http://www.svgjs.com/
+// Definitions by: Kevin Gutiérrez <https://github.com/jkevingutierrez>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Reference: https://github.com/Fuzzyma/svg.resize.js
+
+
+declare namespace svgjs {
+	export interface Element {
```

Comment:
It shouldn't augment the global `Element` it should augment the `Element` used by svgjs. It should works like this:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/svgjs.draggable/svgjs.draggable.d.ts